app/static/win32/github.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
+setlocal
+
+set ELECTRON_RUN_AS_NODE=1
+call "%~dp0..\..\..\GitHub Desktop.exe" "%~dp0..\cli.js" %*
```

Comment:
Yeah, I wasn't sure I cared. I made an attempt at having it work but my batch skills are not up to snuff. If we care, I'll probably need a hand to get this working.